content/en/blog/_posts/2023-09-15-userns-stateful-pods/index.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: blog
+title: "Kuberntes 1.28: Alpha support to run stateful pods too!"
```

Comment:
Try:
```suggestion
title: Run Containers Inside A User Namespace With Kubernetes
```

Reply:
Removed the 1.28 from the title. But that one is too generic IMHO, changed it to: `title: "User namespaces: Now alpha support to run stateful pods too!"`